docs/src/troubleshooting.md

```diff
@@ -63,7 +63,7 @@ checking build system type... Invalid configuration `x86_64-linux-musl': system
 configure: error: /bin/sh ./config.sub x86_64-linux-musl failed
 ```
 
-The `BinaryBuilder` environment provides the utility [`update_configure_scripts`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Utilities-in-the-build-environment-1) to automatically update these scripts, call it before `./configure`:
+The `BinaryBuilder` environment provides the utility [`update_configure_scripts`](@ref utils_build_env) to automatically update these scripts, call it before `./configure`:
```

Comment:
Instead of always linking to the online dev version of the manual, let's use `@ref` to get nice relative links. I've tested this locally.